sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosQueryRequestOptions.java

```diff
@@ -59,6 +59,7 @@ public class CosmosQueryRequestOptions {
     private String queryName;
     private CosmosEndToEndOperationLatencyPolicyConfig cosmosEndToEndOperationLatencyPolicyConfig;
     private List<CosmosDiagnostics> cancelledRequestDiagnosticsTracker = new ArrayList<>();
+    private List<String> excludeRegions;
```

Comment:
I see now in this PR, the thresholdBased policy is open for write operations as well, should we also update the `CosmosBatchPatchItemRequestOptions` and `CosmosBulkItemRequestOptions`?

Reply:
Not handling bulk/batch/patch in this PR